syntax = "proto3";

import "cheers/proto/misc.proto";

package cheers.message;

message PathConfig {
  string name = 1;
  float line_width = 2;
  ColorRGB color = 3;
  Pose sensor_pose = 4;
}

message Path {
  PathConfig config = 1;
  repeated Pose pose = 2;
}
